From 75457ff5f617e280616101fad9940d622de3884d Mon Sep 17 00:00:00 2001
From: Garrett Brown <themagnificentmrb@gmail.com>
Date: Tue, 17 Jan 2023 12:13:29 -0800
Subject: [PATCH] Fix missing library version

---
 src/libretro.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/libretro.cpp b/src/libretro.cpp
index bc6152b222..0a39b2817b 100644
--- a/src/libretro.cpp
+++ b/src/libretro.cpp
@@ -113,7 +113,7 @@ void retro_get_system_info(struct retro_system_info *info) {
 #else
 #define __GIT_VERSION ""
 #endif
-  info->library_version = __GIT_VERSION;
+  info->library_version = SCUMMVM_VERSION;
   info->valid_extensions = "scummvm";
   info->need_fullpath = true;
   info->block_extract = false;
-- 
2.34.1

