From 78382e272d00a0bc4d564cff796c005b925b12e1 Mon Sep 17 00:00:00 2001
From: Garrett Brown <themagnificentmrb@gmail.com>
Date: Fri, 24 Mar 2023 12:33:47 -0700
Subject: [PATCH 2/3] libretro: Move scummvm.ini to writable save directory

---
 src/libretro-os.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/libretro-os.cpp b/src/libretro-os.cpp
index 27a442d..3b52467 100644
--- a/src/libretro-os.cpp
+++ b/src/libretro-os.cpp
@@ -742,7 +742,7 @@ public:
 	}
 
 	virtual Common::String getDefaultConfigFileName() {
-		return s_systemDir + "/scummvm.ini";
+		return s_saveDir + "/scummvm.ini";
 	}
 
 	virtual void logMessage(LogMessageType::Type type, const char *message) {
-- 
2.34.1

