From 4e7b3fc86c0c39ddb376ecb76b22fd816e6af0ee Mon Sep 17 00:00:00 2001
From: Garrett Brown <themagnificentmrb@gmail.com>
Date: Wed, 17 Jul 2024 18:10:03 -0700
Subject: [PATCH 3/3] LIBRETRO: Patch libmad libretro dependency

---
 backends/platform/libretro/dependencies.mk | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/backends/platform/libretro/dependencies.mk b/backends/platform/libretro/dependencies.mk
index c0174491bbd..9c0e7cf0910 100644
--- a/backends/platform/libretro/dependencies.mk
+++ b/backends/platform/libretro/dependencies.mk
@@ -6,8 +6,8 @@
 DEPS_SUBMODULES             := libretro-deps libretro-common
 
 DEPS_FOLDER_libretro-deps   := libretro-deps
-DEPS_URL_libretro-deps      := https://github.com/libretro/libretro-deps
-DEPS_COMMIT_libretro-deps   := 7e6e34f0319f4c7448d72f0e949e76265ccf55a1
+DEPS_URL_libretro-deps      := https://github.com/kodi-game/libretro-deps
+DEPS_COMMIT_libretro-deps   := 37475a099679f25f4c1f35e7656ce2ca72631e6a
 
 DEPS_FOLDER_libretro-common := libretro-common
 DEPS_URL_libretro-common    := https://github.com/libretro/libretro-common
-- 
2.50.1 (Apple Git-155)

