From 8b62e5470311749e9d1ea648a9ab806e9005a578 Mon Sep 17 00:00:00 2001
From: Garrett Brown <themagnificentmrb@gmail.com>
Date: Wed, 17 Jul 2024 18:10:03 -0700
Subject: [PATCH 3/3] LIBRETRO: Patch libmad libretro dependency

---
 backends/platform/libretro/dependencies.mk | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/backends/platform/libretro/dependencies.mk b/backends/platform/libretro/dependencies.mk
index 431d477c95c..91613997fc3 100644
--- a/backends/platform/libretro/dependencies.mk
+++ b/backends/platform/libretro/dependencies.mk
@@ -6,8 +6,8 @@
 DEPS_SUBMODULES             := libretro-deps libretro-common
 
 DEPS_FOLDER_libretro-deps   := libretro-deps
-DEPS_URL_libretro-deps      := https://github.com/libretro/libretro-deps
-DEPS_COMMIT_libretro-deps   := abf5246b016569759e7d1b0ea91bb98c2e34d160
+DEPS_URL_libretro-deps      := https://github.com/kodi-game/libretro-deps
+DEPS_COMMIT_libretro-deps   := 5d0727b3dba4d9da6474a35d7283ba49def32d85
 
 DEPS_FOLDER_libretro-common := libretro-common
 DEPS_URL_libretro-common    := https://github.com/libretro/libretro-common
-- 
2.34.1

